apps/prairielearn/src/ee/pages/institutionAdminLti13/institutionAdminLti13.html.ts

```diff
@@ -339,6 +339,13 @@ ${JSON.stringify(instance.custom_fields, null, 3)}</textarea
       </form>
 
       <hr />
+      <p>
+        For testing, have the LMS admin configure their OpenID Connect Initiation Url to
+        <a href="/pl/lti13_instance/${instance.id}/auth/login?RelayState=test">
+          /pl/lti13_instance/${instance.id}/auth/login?RelayState=test
```

Comment:
I imagine we need a protocol/host in this URL. You and I would know to add that before passing it to an institutional IT person, but it can't hurt to bake it in here. See `instituionAdminSaml` or `lti13Instance` for prior art.

Reply:
Added links in 517b8a1

Reply:
Ahh, I missed that, thanks. Updated in 8e2489f1